src/tests/test_magic.py

```diff
@@ -187,6 +187,11 @@ def test_persist_no_index(ip):
     assert persisted == [(1, "foo"), (2, "bar")]
 
 
+def test_unrecognized_arguments(ip):
+    result = ip.run_cell("%sql --stuff sqlite://")
```

Comment:
we shouldn't apply this to the line magic, only to the cell magic: https://github.com/ploomber/jupysql/issues/531#issuecomment-1571130589